src/lib/iframe-tools/identity-manager.js

```diff
@@ -87,12 +87,27 @@ export class IdentityManager extends IframeManager {
       return;
     }
 
-    debugLogger.log('[identity-manager:fetchIdentity] Fetch country request.');
-    this.fetchCountry().then((country) => {
+    const localStorageKey = 'pp-sdk-shouldReallyCheckCountry';
+    const LS = window.localStorage;
```

Comment:
Can we use sessionStorage, since it has self cleaning mechanism?

Reply:
I thought about that, but we want the `shouldFetchCountry` decision to always persist for a user so that we have a noiseless picture in our graphs. If one day you fetchCountry and the next you don't, it is inconsistent.